driver/razermouse_driver.h

```diff
@@ -72,6 +73,9 @@
 #define RAZER_NEW_MOUSE_RECEIVER_WAIT_MIN_US 31000
 #define RAZER_NEW_MOUSE_RECEIVER_WAIT_MAX_US 31100
 
+#define RAZER_ATHERIS_RECEIVER_WAIT_MIN_US 400000
+#define RAZER_ATHERIS_RECEIVER_WAIT_MAX_US 400100
```

Comment:
wtf :facepalm: 

(you can just 'resolve' this, just wanted to add a comment)

Reply:
YES :rofl: